src/Core/Domain/Product/Stock/Query/GetProductStockMovements.php

```diff
@@ -64,11 +61,6 @@ class GetProductStockMovements
      */
     private $productId;
 
-    /**
-     * @throws ProductConstraintException
-     * @throws ShopException
-     * @throws LogicException
-     */
```

Comment:
unrelated change, just noticed these docs are outdated, constructor doesn't throw any of these exceptions and all arguments are typed so I removed the docblock completely